src/elisp/treemacs-mode.el

```diff
@@ -228,7 +228,11 @@ Will be set by `treemacs--post-command'.")
                '(:eval (moody-tab " Treemacs " 10 'down)))
               ((and (fboundp 'doom-modeline)
                     (fboundp 'doom-modeline-def-modeline))
-               (doom-modeline-def-modeline 'treemacs '(bar " " major-mode))
+               (doom-modeline-def-segment treemacs
+                 "Display treemacs."
+                 (propertize (format " %s " (treemacs-workspace->name (treemacs-current-workspace)) )
```

Comment:
propertize can be removed if no face is provieded.

Reply:
My mistake.. I meant to commit with the face for minor mode